src/Extensions/UI/AssemblyCache.cs

```diff
@@ -0,0 +1,98 @@
+//MIT license
+
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using System.Web.Script;
+
+namespace System.Web.UI;
+
+// Caches Assembly APIs to improve performance
+internal static class AssemblyCache {
```

Comment:
do we need this class? what is it used for?

Reply:
https://referencesource.microsoft.com/#System.Web.Extensions/UI/AssemblyCache.cs,bbfc72b7eb1647dc,references => mostly in the ScriptReference.

Reply:
ok - just seems like a class we may not want to use in the future. I've created an issue to track that